test/crv-staking/incentivized-voting-lockup-gov.ts

```diff
@@ -0,0 +1,1279 @@
+    const s = slope.mul(10000).mul(sqrt(lockupLength));
+    return s;
+  };
+
+  const goToNextUnixWeekStart = async () => {
+    const unixWeekCount = (await getTimestamp()).div(ONE_WEEK);
+    const nextUnixWeek = unixWeekCount.add(1).mul(ONE_WEEK);
+    await increaseTimeTo(nextUnixWeek);
+  };
+
+  const calcBias = async (amount: BN, len: BN): Promise<BN> => {
+    return amount.div(await votingLockup.MAXTIME()).mul(len);
+  };
+
+  const deployFresh = async (initialRewardFunding = BN.from(0)) => {
+    await mta
+      .connect(sa.fundManager.signer)
+      .transfer(
+        sa.default.address,
+        simpleToExactAmount(1000, DEFAULT_DECIMALS)
+      );
+    await mta
+      .connect(sa.fundManager.signer)
+      .transfer(sa.other.address, simpleToExactAmount(1000, DEFAULT_DECIMALS));
+    votingLockup = await IncentivizedVotingLockup.deploy(
+      mta.address,
+      redeemer.address,
+      await mta.name(),
+      await mta.symbol()
+    );
+
+    await votingLockup.deployed();
+    await mta.approve(
+      votingLockup.address,
+      simpleToExactAmount(100, DEFAULT_DECIMALS)
+    );
+    await mta
+      .connect(sa.other.signer)
+      .approve(
+        votingLockup.address,
+        simpleToExactAmount(100, DEFAULT_DECIMALS)
+      );
+    await mta
+      .connect(sa.fundManager.signer)
+      .approve(
+        votingLockup.address,
+        simpleToExactAmount(10000, DEFAULT_DECIMALS)
+      );
+  };
+
+  describe("checking balances & total supply", () => {
+    before(async () => {
+      await deployFresh();
+    });
+    describe("before any stakes are made", () => {
+      it("returns balances", async () => {
+        expect(await votingLockup.staticBalanceOf(sa.default.address)).eq(
+          BN.from(0)
+        );
+        expect(await votingLockup.balanceOf(sa.default.address)).eq(BN.from(0));
+        expect(await votingLockup.balanceOfAt(sa.default.address, 1)).eq(
+          BN.from(0)
+        );
+      });
+      it("returns balance at latest block", async () => {
+        expect(
+          await votingLockup.balanceOfAt(
+            sa.default.address,
+            BN.from((await latestBlock()).number)
+          )
+        ).eq(BN.from(0));
+      });
+      it("returns totalSupply", async () => {
+        expect(await votingLockup.totalSupply()).eq(BN.from(0));
+        expect(await votingLockup.totalSupplyAt(1)).eq(BN.from(0));
+      });
+      it("returns totalSupply at latest block", async () => {
+        expect(
+          await votingLockup.totalSupplyAt(
+            BN.from((await latestBlock()).number)
+          )
+        ).eq(BN.from(0));
+      });
+    });
+    describe("fetching for current block", () => {
+      it("fails for balanceOfAt", async () => {
+        await expect(
+          votingLockup.balanceOfAt(
+            sa.default.address,
+            BN.from((await latestBlock()).number).add(1)
+          )
+        ).to.be.revertedWith("Must pass block number in the past");
```

Comment:
keep on getting **error TS2339: Property 'revertedWith' does not exist on type 'Assertion'.** in all the .to.be.revertedWith errors but setup is same as ribbon-v2. have you run into this?